native/android/app/buildConfigs.gradle

```diff
@@ -28,11 +28,13 @@ def execCommand(command) {
 }
 
 def determineBuildConfigName() {
-    if (!System.getenv()["BUILD_CONFIG_NAME"]) {
+    if (project.hasProperty('BUILD_CONFIG_NAME')) {
+        return project.BUILD_CONFIG_NAME
```

Comment:
Why should the project already have this variable?

Reply:
Oh oke because of the properties in the Fastfile. Hm.. I think we should stick to the environment variable.

Else it is unclear what happens if the environment variable and the project properties variable is set to different values.

Reply:
I rather find it unclear that everything else is passed as property but the build config name not. :thinking: